src/pdfkit.cs

```diff
@@ -509,21 +1321,44 @@ interface PdfDocument : NSCopying {
 
 		[Export ("selectionFromPage:atCharacterIndex:toPage:atCharacterIndex:")]
 		PdfSelection GetSelection (PdfPage startPage, nint startCharIndex, PdfPage endPage, nint endCharIndex);
+
+		[NoiOS]
+		[Mac (10,7)]
+		[Export ("printOperationForPrintInfo:scalingMode:autoRotate:")]
+		[return: NullAllowed]
+		NSPrintOperation GetPrintOperation ([NullAllowed] NSPrintInfo printInfo, PdfPrintScalingMode scaleMode, bool doRotate);
 	}
-	
-	[BaseType (typeof (NSObject))]
+
+	[iOS (11,0)]
+	[BaseType (typeof (NSObject), Name = "PDFDocumentDelegate")]
 	[Model]
-	[Protocol (IsInformal = true)]
+	[Protocol]
```

Comment:
Since forever, but it created issues because it lacked the `Name` attribute to be set so the issue was that `PdfDocumentDelegate` != `PDFDocumentDelegate`